site/content/en/_index.md

```diff
@@ -63,7 +63,7 @@ Runtime indicates which medium `kwokctl` will use to start the cluster
 | Windows | 🟠/🔵    | 🔵      | 🔵      | 🔵       | 🔵    |
 
 - 🟢 Supported
-- 🟠 Need to use your own build of the Kubernetes binary
+- 🟠 Need to use your own build of the [Kubernetes binary without Container]
```

Comment:
the term "without container" doesn't sound that conventional, how about:

```suggestion
- 🟠 Need to build [platform-specific Kubernetes binaries]
```

And we replace "without container" with "platform-specific Kubernetes binaries"